server/server.go

```diff
@@ -74,6 +74,7 @@ func (s *Server) Start() error {
 		if err != nil {
 			return fmt.Errorf("error starting server: %v", err)
 		}
+		runtime.EventsEmit(s.ctx, "onServerStart", s.Id)
```

Comment:
I'd name the event `serverStart`

Reply:
all sevens are called on..... this is kind of the standard rought now in the app